setuptools_odoo/git_file_finder.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2018 ACSONE SA/NV
+# License LGPLv3 (http://www.gnu.org/licenses/lgpl-3.0-standalone.html)
+
+import os
+import subprocess
+
+
+def _git_toplevel(path):
+    try:
+        out = subprocess.check_output([
+            'git', 'rev-parse', '--show-toplevel',
+        ], cwd=(path or '.'), universal_newlines=True)
+        return os.path.realpath(out.strip())
+    except subprocess.CalledProcessError:
+        return None
+
+
+def _git_ls_files_and_dirs(toplevel):
+    out = subprocess.check_output([
+        'git', 'ls-files',
+    ], cwd=toplevel, universal_newlines=True)
+    git_files = {os.path.join(toplevel, f) for f in out.split()}
+    git_dirs = {os.path.dirname(f) for f in git_files}
+    return git_files, git_dirs
+
+
+def find_files(path=''):
+    """ setuptools compatible git file finder that follows symlinks
+
+    Spec here: http://setuptools.readthedocs.io/en/latest/setuptools.html#\
+        adding-support-for-revision-control-systems
+    """
+    toplevel = _git_toplevel(path)
+    if not toplevel:
+        return []
+    git_files, git_dirs = _git_ls_files_and_dirs(toplevel)
+    realpath = os.path.realpath(path)
+    assert realpath.startswith(toplevel)
```

Comment:
@sbidoul Is-it with knowledge that you use `assert`. These assertions will be stripped out and not evaluated if Python is started with the -O option.